cmd/lakefs/cmd/run.go

```diff
@@ -110,7 +111,7 @@ var runCmd = &cobra.Command{
 		authMetadataManager := auth.NewKVMetadataManager(version.Version, cfg.Installation.FixedID, cfg.Database.Type, kvStore)
 		idGen := &actions.DecreasingIDGenerator{}
 
-		// initialize auth service
+		// initialize authorization service
 		var authService auth.Service
```

Comment:
👍 
I think we should rename also other places like the auth package name and authService variable for example
can be done in another PR